scripts/lint-changelog.js

```diff
@@ -125,12 +125,6 @@ for (const category of CHANGELOG_CATEGORIES) {
       );
     }
 
-    if (title.startsWith(" ")) {
```

Comment:
Changelog files are formatted and checked by Prettier, so there can't be a space.